KWZP2019/KWZP2019/DevelopmentHistoryForm.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace KWZP2019
+{
+    public partial class DevelopmentHistoryForm : Form
+    {
+        private RoofingCompanyEntities db;
+        private StartForm startForm;
+        private DevelopmentForm developmentForm;
+        private int selectedDevelopment;
+
+        public DevelopmentHistoryForm(RoofingCompanyEntities db, StartForm startForm, DevelopmentForm developmentForm)
+        {
+            InitializeComponent();
+            this.db = db;
+            this.startForm = startForm;
+            this.developmentForm = developmentForm;
+        }
+
+        private void DevelopmentHistoryForm_FormClosed(object sender, FormClosedEventArgs e)
+        {
+            this.Hide();
+            developmentForm.Show();
+        }
+
+        private void DevelopmentHistoryForm_Load(object sender, EventArgs e)
+        {
+            List<FEMAnalysi> fEMAnalysis = db.FEMAnalysis.ToList();
```

Comment:
nazwa zmiennej powinna wskazywac na to ze jest to kolekcja a nie pojedynczy obiekt

Reply:
@YupiKayey w komentarzu https://github.com/rudnickirobert/KWZP2019/pull/85#discussion_r281071383
chodziło o literkę `s` na końcu- nadal nie poprawione